mlir/tools/mlir-tblgen/OpFormatGen.cpp

```diff
@@ -1310,6 +1315,7 @@ auto setFromAttr = [] (auto &propStorage, ::mlir::Attribute propAttr,
          ::llvm::function_ref<::mlir::InFlightDiagnostic()> emitError) -> ::mlir::LogicalResult {{
   {0};
 };
+usedKeys.insert(StringAttr::get(ctx, "{1}"));
```

Comment:
Is this going to create two StringAttrs? dict.get will likely create one as well, same below

Reply:
Ah, nvm, you won't because dict here is a DictionaryAttr. Probably still faster to share the StringAttr anyways though (get with a StringAttr is faster with small lists)

Reply:
Done, thanks.